src/main/java/de/zalando/nakadi/client/DefaultEventListener2Impl.java

```diff
@@ -0,0 +1,23 @@
+package de.zalando.nakadi.client;
+
+
+import de.zalando.nakadi.client.domain.Cursor;
+import de.zalando.nakadi.client.domain.Event;
+
+public class DefaultEventListener2Impl implements EventListener2 {
```

Comment:
It is the successor of EventListener. At the point in time when I introduced it, I wanted to avoid that this change breaks other clients.